### PR TITLE
Use built versions of packages when deploying pub packages

### PR DESCRIPTION
**Pull request by johnmccutchan**
We now generate .mojom.dart files into the sky_services package. This CL makes big_red_button.py copy the built version of sky* packages into a temporary directory and then publish that.

### CODE REVIEW

File: sky/tools/big_red_button.py [hunk 1]
```diff
@@ -96,9 +110,10 @@ def main():
     pub_path = os.path.join(dart_sdk_root, 'bin/pub')
     android_dist_root = os.path.join(sky_engine_root, 'out/android_Release/dist')
     linux_dist_root = os.path.join(sky_engine_root, 'out/Release/dist')
-    sky_package_root = os.path.join(sky_engine_root, 'sky/packages/sky')
-    sky_engine_package_root = os.path.join(android_dist_root, 'packages/sky_engine/sky_engine')
-    sky_services_package_root = os.path.join(android_dist_root, 'packages/sky_services/sky_services')
+    dart_pkg_root = os.path.join(sky_engine_root, 'out/Release/gen/dart-pkg')
```
Comment (eseidelGoogle) [inline sky/tools/big_red_button.py]: I don't think you want to use androidRelease for one and Release for another.

File: sky/tools/big_red_button.py [hunk 2]
```diff
@@ -73,6 +73,20 @@ def upload_artifacts(dist_root, config, commit_hash):
...
 
+def copy_package(src_dir, dst_dir, ignore=None):
+    # Removes existing destination directory.
+    shutil.rmtree(dst_dir, True)
+    shutil.copytree(src_dir, dst_dir, symlinks=False, ignore=ignore)
+
+
+def publish(pub_path, package_name, package_dir):
+    temp_dir = tempfile.mkdtemp(prefix='sky_pub_packages-')
+    src_dir = package_dir
```
Comment (eseidelGoogle) [inline sky/tools/big_red_button.py]: Thsi local is only used once.

File: sky/tools/big_red_button.py [hunk 3]
```diff
@@ -73,6 +73,20 @@ def upload_artifacts(dist_root, config, commit_hash):
...
+    dst_dir = os.path.join(temp_dir, package)
+    copy_package(src_dir, dst_dir, ignore)
```
Comment (eseidelGoogle) [inline sky/tools/big_red_button.py]: I would add a comment about *why* we copy.

**Comment by eseidelGoogle** [comment]
Other than using the wrong directory, lgtm.

File: sky/tools/big_red_button.py (hunk 2 above)
Comment (johnmccutchan) [inline sky/tools/big_red_button.py]: Done.

**Pull request closed without merging**